package.json

```diff
@@ -44,7 +44,9 @@
   "devDependencies": {
     "@types/node": "18.0.3",
     "@types/react": "18.0.15",
+    "react": "latest",
```

Comment:
`remix-run` complained about `react` being missing, and it was right. If it's in `peerDependencies`, it must be a dev dependency.